stepruncmd/stdout.go

```diff
@@ -0,0 +1,66 @@
+package stepruncmd
```

Comment:
[stepoutputwriter.go](https://github.com/bitrise-io/bitrise/pull/874/files#diff-6ab3eac102b07c604f64929206a3d49acf347f363cc4c08dc33d5b20a5391b8f) content moved here. The only change is that the constructor's parameter is an `io.Writer` and not an `opts log.LoggerOpts`